### PR TITLE
Images re-linked to the site [cars company]

### LINKED ISSUE #4501: [Car company] Home page styling
**Describe the Feature you want to end**
 I would like to add components to the home page styling

**Expected behavior**
 It helps in increasing user experience

**Screenshots (optional)**

**Additional context (optional )**

### PR DESCRIPTION
**Pull request by SnigdhaP2022**
# Car company images linked

Close #4501 

# Checklist:

- [✅] I have mentioned the issue number in my Pull Request.
- [✅] I have commented my code, particularly in hard-to-understand areas

# Before
Screenshot 2023-08-03 at 4 47 37 PM

# After
Screenshot 2023-08-03 at 4 45 22 PM

### CODE REVIEW

**Comment by SnigdhaP2022** [comment]
@TusharKesarwani  Kindly, review this

**Comment by SnigdhaP2022** [comment]
@MohitGupta121 @TusharKesarwani Kindly, review this

File: Frontend-Projects/car company website/images/cars.jpeg
Comment (MohitGupta121) [inline Frontend-Projects/car company website/images/cars.jpeg:1]: @SnigdhaP2022 remove this image from PR and use it as a link after uploading somewhere else.

**Review by MohitGupta121 — CHANGES_REQUESTED** [review]

File: Frontend-Projects/car company website/images/cars.jpeg
Comment (SnigdhaP2022) [reply Frontend-Projects/car company website/images/cars.jpeg:1]: @MohitGupta121 issue has been resolved. Kindly, review it

**Review by MohitGupta121 — APPROVED** [review]

**Pull request merged**